mkosi/__init__.py

```diff
@@ -4848,6 +4848,21 @@ def parse_remove_files(value: str) -> List[str]:
     return ["/" + os.path.normpath(p).lstrip("/") for p in value.split(",") if p]
 
 
+def parse_ssh_agent(value: str) -> Union[Path, bool]:
+    """Can be True/False or a path to a socket."""
+
+    try:
+        if not parse_boolean(value):
+            return False
+    except ValueError:
+        return Path(value)
+
+    sock = os.getenv("SSH_AUTH_SOCK")
+    if sock is None:
```

Comment:
Is it sensible if `SSH_AUTH_SOCK` exists and is empty? If not this might be better served as
```suggestion
    if not sock:
```